x/rollapp/types/rollapp.go

```diff
@@ -108,7 +105,11 @@ func (r Rollapp) ValidateBasic() error {
 }
 
 func (r Rollapp) IsTransferEnabled() bool {
-	return r.GenesisState.TransfersEnabled
+	return r.GenesisState.IsTransferEnabled()
+}
+
+func (s RollappGenesisState) IsTransferEnabled() bool {
```

Comment:
redundent? can use `ra.IsTransferEnabled()` all over instead `ra.GenesisState.IsTransferEnabled()`

Reply:
decline nit, need to ship